PicApp_git/PicApp/public/css/app.css

```
@@ -10883,4 +10883,6 @@ a.text-dark:focus {
     border-color: #dee2e6;
   }
 }
-
+comment-area{
```

クラスに対してのスタイルなので正しくは下記ですね
```suggestion
.comment-area{
```